File: pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
    {
        float owed;
        int owed_cents;
        do
        {
            owed = get_float("Change owed: ");
        } while (owed<=0.00);
        owed_cents = round(owed*100);

        int quarters = floor(owed_cents/25);
        owed_cents = owed_cents - quarters*25;

        int dimes = floor(owed_cents/10);
        owed_cents = owed_cents - dimes*10;

        int nickels = floor(owed_cents/5);
        owed_cents = owed_cents - nickels*5;

        int pennies = floor(owed_cents/1);
        owed_cents = owed_cents - pennies;

        int coins = quarters + dimes + nickels + pennies;
        printf ("%i\n", coins);


    }

File: pset6/cash/cash.py
from cs50 import get_float

owed = 0
owed_change = 0
i = 0

owed = get_float("Owed: ")
while owed <= 0.00:
    owed = get_float("Owed: ")
owed = float(owed)
owed_change = round(owed*100)
while owed_change != 0:
    if owed_change >= 25:
        owed_change = owed_change - 25
    elif 10 <= owed_change < 25:
        owed_change = owed_change - 10
    elif    5 <= owed_change < 10:
        owed_change = owed_change - 5
    else:
        owed_change = owed_change - 1
    i = i + 1
print(i)